code/implementation/host-stats/src/main/java/io/cattle/platform/host/stats/utils/StatsConstants.java

```diff
@@ -0,0 +1,9 @@
+package io.cattle.platform.host.stats.utils;
+
+public class StatsConstants {
+
+    public static final String LINK_STATS = "stats";
```

Comment:
@wlan0 Can you change these to camelCase so that the keys in the link map match the case of all the other links?
Meaning:
`'links': { 'containerStats': 'http...'}` instead of `containerstats`